### PR TITLE
migration of filter translations

### PR DESCRIPTION
**Pull request by Mathieu-Deharbe**
Migrated to https://github.com/gridsuite/commons-ui/pull/559

Therefore this pull request should be merged only after the merging of the commons-ui PR an should use its common-ui last version.

### CODE REVIEW

File: src/components/app-wrapper.tsx [hunk 1]
```diff
@@ -22,6 +22,9 @@ import {
     filter_en,
     filter_expert_en,
     filter_expert_fr,
+    description_fr,
```
Comment (basseche) [inline src/components/app-wrapper.tsx:26]: description_en seem to be missing

File: src/components/app-wrapper.tsx [hunk 2]
```diff
@@ -214,6 +217,9 @@ const messages: Record<GsLangUser, IntlConfig['messages']> = {
         ...directory_items_input_fr,
         ...filter_fr,
         ...filter_expert_fr,
+        ...description_fr,
```
Comment (basseche) [inline src/components/app-wrapper.tsx:226]: description_en seem to be missing, same for equipments_en and csv_en

Capture d’écran du 2024-08-27 09-20-13

File: src/components/app-wrapper.tsx [hunk 3]
```diff
@@ -22,6 +22,9 @@ import {
     filter_en,
     filter_expert_en,
     filter_expert_fr,
+    description_fr,
+    equipments_fr,
+    csv_fr,
```
Comment (basseche) [inline src/components/app-wrapper.tsx:30]: same for equipments_en et csv_en.

File: src/translations/en.json [hunk 4]
```diff
@@ -90,98 +81,42 @@
...
-    "PropertyValues2": "Property values 2",
     "getAppLinkError": "Error getting application link for type = {type}",
-    "nameValidityCheckErrorMsg": "Error while checking name validity",
-    "cantSubmitWhileValidating": "Impossible to submit the form while validating a field",
     "missingEquipmentsIdsError": "ID is required field for all equipments",
-    "distributionKeyWithMissingIdError": "Missing ID with defined distribution key",
     "contingencyTablePartiallyDefinedError": "At least one contingency is partially defined",
     "contingencyTableContainAtLeastOneRowError": "Contingency list should contain at least one contingency ",
-    "missingDistributionKeyError": "Missing distribution key",
     "emptyFilterError": "Filter should contain at least one equipment",
```
Comment (basseche) [inline src/translations/en.json]: You deleted "emptyFilterError" label in FR file but not here. I've checked in commons.ui and found the translation.

File: src/components/app-wrapper.tsx (hunk 1 above)
Comment (Mathieu-Deharbe) [reply src/components/app-wrapper.tsx:26]: Indeed. Added.

File: src/translations/en.json (hunk 4 above)
Comment (Mathieu-Deharbe) [reply src/translations/en.json]: Indee well seen. Removed.

**Review by basseche — APPROVED** [review]
C'est Ok pour moi

**Comment by basseche** [comment]
There is a problem with : CI / build-and-deploy but i suppose because we have to build with the right commons-ui

**Review by ghazwarhili — APPROVED** [review]

**Pull request merged**